### PR TITLE
Log out action

### PR DESCRIPTION
**Pull request by afmicc**

### CODE REVIEW

File: src/api/userApi.js [hunk 1]
```diff
@@ -42,3 +42,16 @@ export const getProfile = async id => {
     handleError(error);
   }
 };
+
+export const logout = async () => {
+  try {
+    const response = await fetch(`${baseUrl}/users/sign_out`, {
+      method: 'DELETE',
+      headers: { 'content-type': 'application/json' }
+    });
```
Comment (lucia-santos) [inline src/api/userApi.js]: As an improvement you can create an `apiService` as it is defined in the `react-redux-base` where you centralize the addition of the `baseUrl` and `headers` to every call. This way you avoid having to:
- import the baseUrl for each api 
- add the headers each time

File: src/redux/reducers/userReducer.js [hunk 2]
```diff
@@ -40,5 +42,14 @@ export default createReducer(intialState, {
...
       'An error occured. Please try again.'
+  }),
+  [logoutSuccess]: () => ({
+    intialState
+  }),
+  [logoutError]: (state, { payload }) => ({
+    ...state,
+    error:
+      (payload && payload.full_messages && payload.full_messages[0]) ||
+      'An error occured. Please try again.'
```
Comment (lucia-santos) [inline src/redux/reducers/userReducer.js]: You can use the optional chaining operator: https://v8.dev/features/optional-chaining
```suggestion
      payload?.full_messages?.[0] || 'An error occurred. Please try again.' 
```

**Review by lucia-santos — APPROVED** [review]
Good job! 👏

File: src/redux/reducers/userReducer.js (hunk 2 above)
Comment (afmicc) [reply src/redux/reducers/userReducer.js]: Nice! I didn't know that `?.[]` exists. Thanks by the extra info link

**Review by agch-dev — APPROVED** [review]

**Pull request merged**